downstream/modules/platform/ref-ports-firewalls-requirements.adoc

```diff
@@ -35,54 +37,61 @@ ALLOW connections from controller(s) to database port
 .{ControllerNameStart}
 [options="header"]
 |===
-|Port |Protocol |Service |Installer Inventory Variable |Required for
+|Port |Protocol |Service |Direction |Installer Inventory Variable |Required for
 |22
 |TCP
 |SSH
-|ansible_port
+|Bi-directional
+|`ansible_port`
 |Installation
 
 |80
 |TCP
 |HTTP
-|nginx_http_port
+|Incoming
+|`nginx_http_port`
 |UI/API
 
 |443
 |TCP
 |HTTPS
-|nginx_https_port
+|Incoming
+|`nginx_https_port`
 |UI/API
 
 |5432
 |TCP
 |PostgreSQL
-|pg_port
+|Bi-directional
+|`pg_port`
 a|Open *only* if the internal database is being used along with another component. Otherwise, this port should not be opened
 
 Hybrid mode in a cluster
 
 |27199
 |TCP
 |Receptor
-|receptor_listener_port
+|Bi-directional
+|`receptor_listener_port`
 |ALLOW receptor port across all controllers (for mandatory & automatic control plane peering)
```

Comment:
```suggestion
|ALLOW the Receptor listener port across all controllers for mandatory and automatic control plane clustering.
```

Reply:
I'm not sure we need "mandatory and automatic" there. My understanding is that port allows Receptor nodes to form clusters. Each Receptor node listens on that port for new joining nodes. It is mandatory in the sense that if you don't allow traffic through the port then nodes won't form a cluster. It's also automatic but I'm not sure that really adds anything here.

Reply:
The stakeholder included those words specifically to allow users of every technical level to understand. I will keep his wording, but revise sentence as you suggested. Thanks!